CHANGELOG.md

```diff
@@ -4,10 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.18.2] - 2024-04-09
+## Unreleased
 ### Fixed
+- Fixed all pedantic lint warnings
+
+## [0.18.3] - 2024-04-09
+### Changed
 - Fixed CI build
 
+## [0.18.2] - 2024-04-09
+### Fixed
+- Updated to `kamu v0.171.2` to correct the CLI push command in the Data access panel
+
```

Comment:
NOTE: In the last PR I had two records stuck together (something I did with git and commits)